docker/build_helper.sh

```diff
@@ -118,5 +119,19 @@ if [ "${BUILD_PYTHON}" = "ON" ]; then
 
         ../dali/python/bundle-wheel.sh nvidia_dali[_-]*.whl
     fi
-fi
 
+    if [ "${BUILD_DUMMY_PKG}" = "ON" ]; then
+        mkdir -p dummy_build
+        pushd dummy_build
+        export CUDA_VERSION_STR=$(echo $(ls /usr/local/cuda/lib64/libcudart.so*)  | sed 's/.*\.\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)/\1.\2/')
+        cmake ../../dali/python/dummy \
+              -DCUDA_VERSION:STRING="${CUDA_VERSION_STR}" \
+              -DDALI_BUILD_FLAVOR=${NVIDIA_DALI_BUILD_FLAVOR} \
+              -DTIMESTAMP=${DALI_TIMESTAMP} \
+              -DGIT_SHA=${GIT_SHA}
+        python setup.py sdist
+        mkdir -p /wheelhouse/dummy
+        mv dist/*.tar.gz /wheelhouse/dummy
+        popd
+    fi
+fi
```

Comment:
This is really similar to `make_nvidia_dali_dummy.sh`. Can you deduplicate somehow? Are both needed?

Reply:
Will do